app/views/warehouse_reports/chronic/_sort.haml

```diff
@@ -11,8 +11,8 @@
           - else
             - title = "<span class='icon-checkbox-unchecked'></span> #{opt[:title]}".html_safe
           %li
-            - # FIXME replace permit! with a whitelist and remove from brakeman.ignore
-            = link_to title, params.permit!.merge!(direction: opt[:direction], column: opt[:column])
+            = link_to({direction: opt[:direction], column: opt[:column]}) do
```

Comment:
This also appears to lose any previous filter/pagination.

Reply:
Sorting is the only option on the potentially chronic clients report.